src/main/java/net/dries007/tfc/compat/jei/category/GlassworkingRecipeCategory.java

```diff
@@ -71,23 +60,8 @@ public void setRecipe(IRecipeLayoutBuilder builder, GlassworkingRecipe recipe, I
         {
             var slot = builder.addSlot(RecipeIngredientRole.CATALYST, idx < 3 ? 6 : 90, 25 * ((idx % 3) + 1))
                 .setBackground(this.slot, -1, -1);
-            if (map.containsKey(operation))
-            {
-                slot.addItemStack(map.get(operation));
-                if (map.get(operation).getItem() == TFCItems.BLOWPIPE_WITH_GLASS.get())
-                {
-                    slot.addItemStack(TFCItems.CERAMIC_BLOWPIPE_WITH_GLASS.get().getDefaultInstance());
-                }
-            }
-            else if (operation != GlassOperation.TABLE_POUR && operation != GlassOperation.BASIN_POUR)
-            {
-                for (Map.Entry<Item, GlassOperation> entry : GlassOperation.POWDERS.get().entrySet())
-                {
-                    if (entry.getValue() == operation)
-                    {
-                        slot.addItemStack(entry.getKey().getDefaultInstance());
-                    }
-                }
```

Comment:
So, this logic performed a few different useful JEI functions but appears to have been removed. Confirm that the previous functionality is preserved. I haven't looked deeply into it so that may be the case already.